### PR TITLE
Added header for Popups->Issues->Add Issue popup

### LINKED ISSUE #819: Missing issue form header
Go to Popups->Issues-> click Add Issue. The form has no header. Give it a header. 
Click on Add Encounter, the form has a header. Give "Add issue" its header.

### PR DESCRIPTION
**Pull request by Jdew192837**
Also reorganized code for better readability

Fixes #819

### CODE REVIEW

**Comment by Jdew192837** [comment]
Before:
=====
addissuebefore

After:
====
addissueafter

**Comment by Trodrige** [comment]
More eyes here. It looks good to me. Awesome @Jdew192837

File: interface/patient_file/summary/add_edit_issue.php [hunk 1]
```diff
@@ -626,181 +626,218 @@ function divclick(cb, divid) {
...
- onsubmit='return validate()'>
+<form
+  method='post'
+  name='theform'
+  action='add_edit_issue.php?issue=<?php echo attr($issue); ?>&thispid=<?php echo attr($thispid); ?>&thisenc=<?php echo attr($thisenc); ?>'
+  onsubmit='return validate()'
+>
 
 <table border='0' width='100%'>
+  <h1>Add Issue</h1>
```
Comment (teryhill) [inline interface/patient_file/summary/add_edit_issue.php]: You need to add the translation variable to this.

**Comment by Jdew192837** [comment]
Done @teryhill :)

**Comment by aethelwulffe** [comment]
Question, what does EDIT issue look like?
-Applying my rule now:
Since you added a line with the title...I suggest you compact down a line.
Begin and end dates can go on the same line.

**Comment by tmccormi** [comment]
Are we holding this for the changes requested by @aethelwulffe

**Comment by teryhill** [comment]
@Jdew192837  did you see @aethelwulffe coments above?

**Comment by teryhill** [comment]
@Jdew192837 did you see @aethelwulffe coments above?

File: interface/patient_file/summary/add_edit_issue.php [hunk 2]
```diff
@@ -626,181 +626,218 @@ function divclick(cb, divid) {
...
- onsubmit='return validate()'>
+<form
+  method='post'
+  name='theform'
+  action='add_edit_issue.php?issue=<?php echo attr($issue); ?>&thispid=<?php echo attr($thispid); ?>&thisenc=<?php echo attr($thisenc); ?>'
+  onsubmit='return validate()'
+>
 
 <table border='0' width='100%'>
+  <h1><?php echo xlt('Add Issue'); ?></h1>
```
[the remark this answers is not in the document]
Comment (tmccormi) [reply interface/patient_file/summary/add_edit_issue.php]: @Jdew192837 please confirm my question above.

[the remark this answers is not in the document]
Comment (Jdew192837) [reply interface/patient_file/summary/add_edit_issue.php]: Yes @tmccorni

[the remark this answers is not in the document]
Comment (Jdew192837) [reply interface/patient_file/summary/add_edit_issue.php]: @tmccormi

**Pull request merged**